internal/service/handler.go

```diff
@@ -29,8 +30,11 @@ const (
 	routeVarFeatureID = "fid"
 )
 
-func initRouter() *mux.Router {
-	router := mux.NewRouter().StrictSlash(true)
+func initRouter(basePath string) *mux.Router {
+	router := mux.NewRouter().
+	StrictSlash(true).
+	PathPrefix("/" + strings.TrimLeft(basePath, "/")).
```

Comment:
Any need to check for trailing slashes?

Reply:
I initially thought that we should keep trailing slashes, but it might be a good idea to trim them off unless someone has a good use case to keep them.

Reply:
Actually now that I'm testing it out, if I trim the trailing slashes off, the muxRouter is being a little weird.
If I set BasePath to a valid route like `/foo/` it works just fine and allows requests to `/foo` and `/foo/`. However if there is an invalid route such as `/foo//` it still allows requests but just redirects to `/foo/`. I'm not sure if we would want that behavior

Reply:
We should sanitize the supplied basePath by removing all trailing slashes, and then adding a single one if needed.